x/gov/simulation/operations.go

```diff
@@ -587,21 +594,26 @@ func randomDeposit(
 
 	threshold := minDepositAmount.ToLegacyDec().Mul(minDepositRatio).TruncateInt()
 
-	minAmount := sdkmath.ZeroInt()
-	if useMinAmount {
+	minInitAmt := sdkmath.ZeroInt()
+	if useMinInitAmt {
```

Comment:
can you explain this?

Reply:
Yeah, I was trying to keep the original code as much as possible. I made these changes to get the proposal to immediately enter the voting period. So the deposit amount has to satisfy both the "minimum initial deposit requirement" and the "minimum deposit requirement".